src/OnboardingSPA/data/routes/ecommerce-flow.js

```diff
@@ -37,11 +38,11 @@ export const ecommerceSteps = [
 		title: __( 'Street Address', 'wp-module-onboarding' ),
 		heading: __( 'Street Address', 'wp-module-onboarding' ),
 		subheading: __(
-			'In this step you confirm the business address of your store. Simply confirm the one you provided during your initial  Bluehost account setup or provide a new one.',
+			'In this step you confirm the business address of your store. Simply confirm the one you provided during your initial Bluehost account setup or provide a new one.',
 			'wp-module-onboarding'
 		),
 		description: __(
-			'In this step you confirm the business address of your store. Simply confirm the one you provided during your initial  Bluehost account setup or provide a new one.',
+			'In this step you confirm the business address of your store. Simply confirm the one you provided during your initial Bluehost account setup or provide a new one.',
```

Comment:
I think the brand name will need to be dynamic right ?

Reply:
I don't remember seeing this text anywhere in our flow, most probably unused. I'm was going to create a story to move `heading` `subheading` and `description` away from our route files (which get placed into Redux) to lighter content.js files like `src/OnboardingSPA/data/routes/default-flow.js`  in https://github.com/newfold-labs/wp-module-onboarding/pull/226/files. We can remove these old unused keys as part of that.